### PR TITLE
applovin/mrec_zone_and_open_bidding_watermark_support

### PR DESCRIPTION
**Pull request by Vedant-Mehta39**
1. Adds support for Medium Rectangle Ads 
2. Adds support for watermarks when bidding

### CODE REVIEW

**Review by thomasmso — APPROVED** [review]

**Comment by joshh-devrel** [comment]
@thomasmso Can you rebase this CL? We just did a version bump for 11.8.2.0. Thanks!

**Comment by Vedant-Mehta39** [comment]
@joshh-devrel I can't seem to find the version bump to 11.8.2.0 on master.

**Comment by joshh-devrel** [comment]
@Vedant-Mehta39 Can you try to rebase again? Thanks!

**Comment by Vedant-Mehta39** [comment]
@joshh-devrel I have rebased the branch. Do let me know if I need to make any other changes

File: adapters/AppLovin/AppLovinAdapter/Bidding/GADMRTBAdapterAppLovinInterstitialRenderer.m [hunk 1]
```diff
@@ -73,6 +73,7 @@ - (void)loadAd {
 
   // Create interstitial object.
   _interstitialAd = [[ALInterstitialAd alloc] initWithSdk:_SDK];
+  [_interstitialAd setExtraInfoForKey:@"google_watermark" value:_adConfiguration.watermark];
```
Comment (joshh-devrel) [inline adapters/AppLovin/AppLovinAdapter/Bidding/GADMRTBAdapterAppLovinInterstitialRenderer.m:76]: @Vedant-Mehta39 
As `_adConfiguration.watermark` is `NSData` type, can you confirm type conversion isn't needed before doing `setExtraInfoForKey`?

File: adapters/AppLovin/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 #### Version 11.8.2.0 (In Progress)
 - Verified compatibility with AppLovin SDK 11.8.2.
+- Added support for watermark when bidding.
```
Comment (joshh-devrel) [inline adapters/AppLovin/CHANGELOG.md:5]: Nit, `Added watermark support for bidding ads.`

File: adapters/AppLovin/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
+- Added support for Medium Rectangle(MREC) ads.
```
Comment (joshh-devrel) [inline adapters/AppLovin/CHANGELOG.md:6]: Nit, `Added support for MREC size banner ads.`

File: adapters/AppLovin/AppLovinAdapter/Bidding/GADMAdapterAppLovinRewardedRenderer.m [hunk 4]
```diff
@@ -83,6 +83,8 @@ - (void)requestRewardedAd {
 
   // Create rewarded video object.
   _incent = [[ALIncentivizedInterstitialAd alloc] initWithSdk:_SDKk];
+  [_incent setExtraInfoForKey:@"google_watermark" value:_adConfiguration.watermark];
+
```
Comment (joshh-devrel) [inline adapters/AppLovin/AppLovinAdapter/Bidding/GADMAdapterAppLovinRewardedRenderer.m:87]: Our system detects there are empty spaces on this line, can you remove them?

File: adapters/AppLovin/AppLovinAdapter/Bidding/GADMRTBAdapterAppLovinInterstitialRenderer.m (hunk 1 above)
Comment (Vedant-Mehta39) [reply adapters/AppLovin/AppLovinAdapter/Bidding/GADMRTBAdapterAppLovinInterstitialRenderer.m:76]: Yes, type conversion isn't needed

**Comment by Vedant-Mehta39** [comment]
Changes Made

**Comment by Vedant-Mehta39** [comment]
Changes Made:

Deleted AppLovinRtbBannerRenderer and removed it's usages

**Pull request merged**